zebrad/src/components/sync.rs

```diff
@@ -66,13 +73,6 @@ const MAX_CHECKPOINT_DOWNLOAD_SECONDS: u64 = 300;
 /// then without a timeout, Zebra would deadlock.
 const BLOCK_VERIFY_TIMEOUT: Duration = Duration::from_secs(MAX_CHECKPOINT_DOWNLOAD_SECONDS);
 
-/// Controls how long we wait to retry ExtendTips after it fails.
-///
-/// This timeout should be long enough to allow some of our peers to clear
-/// their connection state.
-///
-/// (ObtainTips failures use the sync restart timeout.)
-const TIPS_RETRY_TIMEOUT: Duration = Duration::from_secs(60);
 /// Controls how long we wait to restart syncing after finishing a sync run.
 ///
 /// This timeout should be long enough to:
```

Comment:
This comment needs updating, but I don't know enough about the details of #1041 and #1089 to make an accurate suggestion.